.github/workflows/make_release.yml

```diff
@@ -41,27 +43,17 @@ jobs:
           fi
           echo "tag=${TAG}" >> $GITHUB_ENV
           # https://help.github.com/en/actions/reference/workflow-commands-for-github-actions
-          echo "::set-output name=contents::$RELEASE_NOTES"
+          echo "name=contents=${RELEASE_NOTES}" >> $GITHUB_ENV
       - name: Create Release
-        id: create_release
-        uses: actions/create-release@latest
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }} # This token is provided by Actions, you do not need to create your own token
+        uses: "softprops/action-gh-release@v1"
         with:
           tag_name: ${{ github.ref }}
-          release_name: ${{ env.tag }}
+          name: ${{ env.tag }}
           body: ${{ steps.release_notes.outputs.contents }}
           draft: false
           prerelease: ${{ contains(github.ref, 'rc') }}
-      - name: Upload Release Asset
-        uses: actions/upload-release-asset@v1
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        with:
-          upload_url: ${{ steps.create_release.outputs.upload_url }}
-          asset_path: ./dist/napari-${{ env.tag }}.tar.gz
-          asset_name: napari-${{ env.tag }}.tar.gz
-          asset_content_type: application/gzip
+          files: |
+            dist/*
       - name: Publish PyPI Package
         uses: pypa/gh-action-pypi-publish@master
         with:
```

Comment:
Do you want to update that to use trusted publisher while at it for 4.19.0 finale ?
https://blog.pypi.org/posts/2023-04-20-introducing-trusted-publishers/

Reply:
I do not have access to pypi account. On project where I administrate I have done it. I think that it is briliant idea but require someone with pypi access.

Reply:
@jni or @kne42 ? I also think maybe you should get PyPI access seeing you are the release manager. You might need to yank a release.